tasks/section2.yml

```diff
@@ -330,10 +328,9 @@
       - rule_2.2.6
 
 - name: "SCORED | 2.2.6 | PATCH | Ensure LDAP server is not enabled"
-  service:
-      name: slapd
-      state: stopped
-      enabled: no
+  yum:
+      name=openldap-servers
+      state=absent
   when: slapd_server_enabled_audit.failed and rhel7cis_ldap_server == false
   tags:
       - level1
```

Comment:
The fix for #7 is to remove line 320 from the original code. `register: result` Since the AUDIT task was incorrectly trying to register the result of the task output in two separate variables the first variable errors out with `'slapd_server_enabled_audit' is undefined`. I removed `register: result` and the AUDIT and PATCH tasks work as intended.

Also the PATCH item is to make sure LDAP server is not enabled and I don't particularly like using the more brute force method of just removing the package to accomplish this.